static/css/landing-page.css

```diff
@@ -112,7 +112,7 @@ h6 {
 .contact {
     padding: 100px 0;
     color: #f8f8f8;
-    background: url(../img/contact-bg.jpg) no-repeat center center;
+    background: url(/img/contact-bg.jpg) no-repeat center center;

```

Comment:
Hello, your PR looks good.
However, I only think this url should be remained as relative path because hugo's canonifyUrls option doesn't work at urls in css files 
so that this will cause a problem if you deploy the site not on a root path (e.g. http://example.com/foo/)